### PR TITLE
NUnit console.

### PR DESCRIPTION
**Pull request by unickq**
CLI tool for dotnet testing

### CODE REVIEW

File: bucket/nunit [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    "suggest": {
+        "NuGet": [
+            "nuget"
+        ]
+    },
+    "checkver": {
+        "github": "https://github.com/nunit/nunit-console"
+    },
+    "autoupdate": {
+	       "url": "https://github.com/nunit/nunit-console/releases/download/v3.9/NUnit.Console-$version.zip"
```
Comment (r15ch13) [inline bucket/nunit]: Replace `/v3.9/` with `/v$majorVersion.$minorVersion/

**Comment by r15ch13** [comment]
The autoupdate URL still required a changed.
Replace `/v3.9/` with `/v$majorVersion.$minorVersion/`

**Comment by unickq** [comment]
@r15ch13 
Done :)

**Comment by Ash258** [comment]
I don't think that autoupdate will work, since you need to replace tag with $major and $minor

**Comment by Ash258** [comment]
This version work: 

```json
"checkver": {
   "github": "https://github.com/nunit/nunit-console/",
   "regex": "NUnit\\.Console-([\\d\\.]+)\\.zip"
}
```

**Comment by unickq** [comment]
@Ash258 
So is this one correct?
```
 "checkver": {
        "github": "https://github.com/nunit/nunit-console/",
        "regex": "NUnit\\.Console-([\\d\\.]+)\\.zip"
    },
    "autoupdate": {
        "url": "https://github.com/nunit/nunit-console/releases/download/v$majorVersion.$minorVersion/NUnit.Console-$version.zip"
    }
```

**Comment by Ash258** [comment]
Keep the autoupdate, just replace checkver.

**Comment by Ash258** [comment]
Again hardcoded tag??

**Comment by unickq** [comment]
@Ash258 
Misunderstood you. Keeping this

```
"checkver": {
        "github": "https://github.com/nunit/nunit-console/",
        "regex": "NUnit\\.Console-([\\d\\.]+)\\.zip"
    },
    "autoupdate": {
        "url": "https://github.com/nunit/nunit-console/releases/download/v$majorVersion.$minorVersion/NUnit.Console-$version.zip"
    }
```

**Pull request merged**